src/components/patient/AddressPanel.tsx

```diff
@@ -28,6 +54,9 @@ const AddressPanel: FC<{ data: PatientRecord }> = ({ data }) => {
         rows={[data]}
         autoHeight={true}
         hideFooter={true}
+        processRowUpdate={(newRow, oldRow) => onRowUpdate(newRow, oldRow)}
```

Comment:
```suggestion
        processRowUpdate={(newRow) => onChange(newRow)}
```

Reply:
@marrouchi `onChange` needs to return `newRow` so that it could be displayed on the data grid